MonoGame.Framework/GameWindow.cs

```diff
@@ -16,12 +16,10 @@ public abstract class GameWindow {
 
 		public abstract Rectangle ClientBounds { get; }
 
-	    internal bool _allowAltF4 = true;
-
         /// <summary>
         /// Gets or sets a bool that enables usage of Alt+F4 for window closing on desktop platforms. Value is true by default.
         /// </summary>
-        public virtual bool AllowAltF4 { get { return _allowAltF4; } set { _allowAltF4 = value; } }
+        public virtual bool AllowAltF4 { get; set; }

```

Comment:
We can rename it to DisableAltF4 and don't worry about setting it on every constractor while also keep it at minimal code as in this PR without the extra field.